app/webpacker/components/RegistrationsV2/api/registration/get/get_registrations.js

```diff
@@ -1,12 +1,11 @@
-import { wcaRegistrationUrl } from '../../../../../lib/requests/routes.js.erb';
+import { getPsychSheetForEventUrl, wcaRegistrationUrl } from '../../../../../lib/requests/routes.js.erb';
 import fetchWithJWTToken from '../../../../../lib/requests/fetchWithJWTToken';
 import { fetchJsonOrError } from '../../../../../lib/requests/fetchWithAuthenticityToken';
 
 const confirmedRegistrationsUrl = (competitionId) => `${wcaRegistrationUrl}/api/v1/registrations/${competitionId}`;
-const getPsychSheetForEventUrl = (competitionId, eventId, sortBy) => `${wcaRegistrationUrl}/api/v1/psych_sheet/${competitionId}/${eventId}?sort_by=${sortBy}`;
 const allRegistrationsUrl = (competitionId) => `${wcaRegistrationUrl}/api/v1/registrations/${competitionId}/admin`;
 const singleRegistrationUrl = (competitionId, userId) => `${wcaRegistrationUrl}/api/v1/register?user_id=${userId}&competition_id=${competitionId}`;
-const waitingCompetitiorsUrl = (competitionId) => `${wcaRegistrationUrl}/api/v1/registrations/${competitionId}/waiting`;
+const waitingCompetitorsUrl = (competitionId) => `${wcaRegistrationUrl}/api/v1/registrations/${competitionId}/waiting`;
```

Comment:
This seems to be inconsistent with the notion of listing some RegMicroservice URLs in the `routes.js.erb` file in the previous PR

Reply:
Not sure which your mean, there were none in the previous PR only the base url

Reply:
Ah, okay!